video/vdu_layers.h

```diff
@@ -10,7 +10,8 @@
 // #include "vdu_layers.h"
 // in vdu_sys.h and is called by VDUStreamProcessor::vdu_sys_video()
 
-#define VDP_LAYERS							0xC2		// VDU 23,0,194
```

Comment:
this has been moved into `agon.h` to the block of code where all the other VDU 23,0 command numbers are defined